unified-codes/libs/ui/src/lib/components/icons/molecules/TMFLogo.tsx

```diff
@@ -36,24 +36,24 @@ export const TMFLogo: SvgIcon = (props) => {
       />
       <text
         fill="#ccc"
-        font-family="Museo"
-        font-size="72"
+        fontFamily="Museo"
```

Comment:
Was this related to the `.env` findings or a bonus fix?

Reply:
hehe. can't get away with anything. 
Yes - offroad, I noticed a few errors in the console. Still one more to fix, will do that next time